src/main/java/macrobase/analysis/classify/StaticThresholdClassifier.java

```diff
@@ -0,0 +1,31 @@
+package macrobase.analysis.classify;
+
+import macrobase.analysis.result.OutlierClassificationResult;
+import macrobase.conf.MacroBaseConf;
+import macrobase.conf.MacroBaseDefaults;
+import macrobase.datamodel.Datum;
+
+import java.util.Iterator;
+
+public class StaticThresholdClassifier extends OutlierClassifier {
+
+    private Double threshold;
+    private Double thresholdSquared;
+
+    public StaticThresholdClassifier(MacroBaseConf conf, Iterator<Datum> input) {
+        super(conf, input);
+        threshold = conf.getDouble(MacroBaseConf.OUTLIER_STATIC_THRESHOLD, MacroBaseDefaults.OUTLIER_STATIC_THRESHOLD);
+        thresholdSquared = threshold * threshold;
+    }
+
+    @Override
+    public boolean hasNext() {
+        return input.hasNext();
+    }
+
+    @Override
+    public OutlierClassificationResult next() {
+        Datum nextInput = input.next();
+        return new OutlierClassificationResult(nextInput, thresholdSquared > nextInput.getMetrics().getNorm());
```

Comment:
> If so, consider re-naming to make that distinction more clear.

A core goal of this refactor is to get rid of the idea of a "score" at all. Metrics are all that classifiers operate over now.